sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsClient.cs

```diff
@@ -766,6 +760,34 @@ private Response<RecognizeEntitiesResultCollection> RecognizeEntitiesBatch(Analy
             }
         }
 
+        private static AnalyzeTextEntityRecognitionInput DocumentsToEntityRecognition(IEnumerable<string> documents, AnalyzeTextEntityRecognitionInput textEntityInputs, TextAnalyticsRequestOptions options, string language)
```

Comment:
Why do you pass a newly created object `AnalyzeTextEntityRecognitionInput textEntityInputs` to then populate it and return it?
Why not just create it here in this function and then return it?

Reply:
I think I did the same thing in LanguageDetection, I'll update that method as well